### PR TITLE
Include changeset id in deletion

### LINKED ISSUE #34: Changeset is not stored on deleted elements
Failing test on osm-p2p-server: https://github.com/digidem/osm-p2p-server/blob/master/test/api/get_changes.test.js#L40-L52

### PR DESCRIPTION
**Pull request by gmaclennan**
@noffle could you review and add a test here (see the failing test in osm-p2p-server https://github.com/digidem/osm-p2p-db/compare/changeset-deletions) attempts to fix #34

### CODE REVIEW

**Comment by noffle** [comment]
What do you think about just having `fields.v` set to `opts.value` directly? This makes it more versatile for the future.

**Comment by gmaclennan** [comment]
That makes sense to me. I'm not sure about @substack's original motivation for choosing this API.

**Comment by noffle** [comment]
Can you also update the `osm.del()` calls we make to set `opts.value.changeset`?

File: index.js [hunk 1]
```diff
@@ -172,6 +172,9 @@ DB.prototype._del = function (key, opts, cb) {
         fields.refs.push.apply(fields.refs, v.refs)
       }
     })
+    if (opts.value && opts.value.changeset) {
```
Comment (noffle) [inline index.js]: How about just checking for `opts.changeset`?

**Comment by noffle** [comment]
Let's nix this PR and update `osm-p2p-db#del()` to accept a `value` on deletions, just like PUTs, like we discussed in meatspace.

**Comment by gmaclennan** [comment]
closing in favour of https://github.com/digidem/osm-p2p-db/pull/46

**Pull request closed without merging**